reco_utils/evaluation/python_evaluation.py

```diff
@@ -677,19 +674,11 @@ def logloss(
         rating_true, rating_pred, col_user, col_item, col_rating, col_prediction
     )
 
-    # To make sure that there is no N/A in the logloss metric calculation.
-    rating_true_pred[PREDICTION_COL] = (
-        rating_true_pred[PREDICTION_COL].apply(lambda x: max(min(x, 1.0 - 10e-12), 10e-12))
-    )
```

Comment:
Why did you remove this completely? It is good to clip the predictions. But you can do it more simply with numpy.clip.

Reply:
Didn't I leave any comment that explain this somewhere?... Actually the `sklearn.metrics.log_loss` take care of the clipping.

Reply:
Great then.